src/v/kafka/server/handlers/configs/config_response_utils.cc

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
> See previous commits in the series for details.

copy and paste anything here that would be needed to understand this in isolation. for example, this wouldn't work in a `git blame` situation.